Resources/Prototypes/Entities/Clothing/OuterClothing/vests.yml

```
@@ -2,7 +2,7 @@
 
 #Hazard vest
 - type: entity
-  parent: [ ClothingOuterStorageBase, AllowSuitStorageClothing ] # Moffstation - I literally just want to be a vox with a hi-viz on instead of a tank harness
+  parent: ClothingOuterStorageSuitStorageAttachableBase # Moffstation - I literally just want to be a vox with a hi-viz on instead of a tank harness
```

```suggestion
  parent: ClothingOuterStorageSuitStorageAttachableBase # Moffstation - Allow OuterClothing suit storage attachment
```
funny but Unhelpfull